packages/arb-ts/token_lists/generate_json.ts

```diff
@@ -106,29 +155,25 @@ const gen = async () => {
     const l2Address = await bridge.getERC20L2Address(l1Address)
     const l2AddressLowerCase = l2Address.toLowerCase()
     const code = await bridge.l2Bridge.l2Provider.getCode(l2Address)
-    if (code.length <= 2)
-      throw new Error(
+    if (code.length <= 2) {
+      console.log(
         `${l1Address} registered at (or defaults to) ${l1GatewayAddress} but not yet deployed on L2`
       )
+      continue
+    }
+
     const arbToken = await StandardArbERC20__factory.connect(
       l2Address,
       bridge.l2Bridge.l2Provider
     )
 
-    const l1Name =
-      l1Address === '0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2'
-        ? 'Maker'
-        : await l1TokenContract.name()
-    const l1Symbol =
-      l1Address === '0x9f8F72aA9304c8B593d555F12eF6589cC3A579A2'
-        ? 'MKR'
-        : await l1TokenContract.symbol()
+    const l1Name = await getL1Name(l1TokenContract)
+    const l1Symbol = await getL1Symbol(l1TokenContract)
     const l1Decimals = await l1TokenContract.decimals()
-
     const l2Symbol =
```

Comment:
if we refactor the getSymbol logic to use `parseBytes32String`, then it should work here too